experiments/can-we-drive-traffic-from-wdtk-to-asktheeu.md

```diff
@@ -0,0 +1,28 @@
+# Can we drive traffic from WDTK to AskTheEU?
+
+We wanted to know if adding adverts with info about and a link to AsktheEU.org to WhatDoTheyKnow would drive traffic to AsktheEU. E.g. Are WDTK users interested in requesting information from EU institutions?
+
+- [issue#50](https://github.com/mysociety/alaveteli-experiments/issues/50)
+
+## Answer
+
+**TL;DR:** Yes.
+
+After putting [this advert](https://cloud.githubusercontent.com/assets/739624/15118227/319e97ec-1603-11e6-9c11-56c3bcad423d.png) on WhatDoTheyKnow's homepage, at the bottom of all request pages, and on [this page](https://www.whatdotheyknow.com/select_authority), we found that:
```

Comment:
[opportunity to disagree with reviewer] I think it might look nice if we embedded the advert image on the page rather than linking out to it. As a bonus feature we'd also get to have the image in our own repo!